package.json

```diff
@@ -116,6 +116,7 @@
     "@react-bootstrap/react-popper": "1.2.1",
     "@restart/context": "^2.1.4",
     "@restart/hooks": "^0.2.3",
+    "astroturf": "^0.9.4",
```

Comment:
why are you adding these dependencies here? they are defined in the website's `package.json`.

https://github.com/react-bootstrap/react-bootstrap/blob/84f0f1b39519f4218665d27b6a4cefc20c651466/www/package.json#L17

https://github.com/react-bootstrap/react-bootstrap/blob/84f0f1b39519f4218665d27b6a4cefc20c651466/www/package.json#L47

Reply:
The linting throw errors and escaped because of these missing dependencies. You are right they are in the `www/package.json` but as I said the linter would not even complete because of 'missing dependencies`.